assets/js/components/user-input/UserInputQuestionWrapper.js

```diff
@@ -104,12 +108,20 @@ export default function UserInputQuestionWrapper( props ) {
 	);
 }
 
+UserInputQuestionWrapper.defaultProps = {
+	backLabel: __( 'Back', 'google-site-kit' ),
+	nextLabel: __( 'Next', 'google-site-kit' ),
```

Comment:
@asvinb let's add it into markup as a fallback values ( `{ nextLabel || __( 'Next', 'google-site-kit' ) }` ) to make sure translations are loaded by the time we render those buttons.

Reply:
Updated @eugene-manuilov